runelite-client/src/test/java/net/runelite/client/plugins/hiscore/HiscorePanelTest.java

```diff
@@ -31,7 +31,6 @@
 	@Test
 	public void testConstructor()
 	{
-		new HiscorePanel(new HiscoreConfig() {});
+		new HiscorePanel(new HiscoreConfig() { });

```

Comment:
This change should not be in this commit/pr

Reply:
Reverted in second commit.